app/views/layouts/_expiration_notice.html.erb

```diff
@@ -5,17 +5,29 @@
   <div class="expiration-content">
     <div class="expiration-head-panel">
       <h1 class="expiration-title"><%= t("expiration.title") %></h1>
-      <h2 class="expiration-subtitle" ><%= t("expiration.sub_title") %></h2>
+      <h2 class="expiration-subtitle" >
+        <% if feature_enabled?(:new_plan_page) %><%= t("expiration.sub_title_new") %>
+        <% else %><%= t("expiration.sub_title") %>
+        <% end %>
+      </h2>
+      <% if feature_enabled?(:new_plan_page) %>
```

Comment:
N.B. I'm not sure how these should work with "APP_CONFIG.external_plan_service_in_use" flag